scripts/prepare_discard_plate.py

```diff
@@ -51,6 +51,7 @@ def has_workflow_stage(artifact, workflow_step_name):
 
 
 class FindPlateToRoute(StepEPP):
+    _use_load_config = False
```

Comment:
This needs the discussed changes in `EPPs.common.StepEPP` in order to be meaningful. This PR could then close #67

Reply:
I guess either Tim or yourself will implement this change?